### PR TITLE
Update blogs.yml

### PR DESCRIPTION
**Pull request by MomtahinaKarim**
Please read and understand everything below
**Do not delete any text other than where you are instructed.**

**Students: If one of them is applicable to you. Please check it.**

Check by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [ ] Read and understood (see CONTRIBUTING.md)
- [ ] Included a Preview link and screenshot showning after and before the changes.
- [ ] Images are `240 x 240` [w x h].
- [ ] Included a description of change below.
- [ ] Squashed the commits.

# Changes done in this Pull Request

- If your change will be reflected on the website, please provide a **Test-Link** (**Hint : `gh-pages`**)
- Fixes`#<Issue no.>`

### CODE REVIEW

File: _data/blogs.yml [hunk 1]
```diff
@@ -3,10 +3,10 @@
 # This is to remove conflicts with other people adding entries.
 
 # The Format is :
-# - url: URL to the blog
-#   title: Title of the blog
-#   img: Image of the blog
-#   author_name: Name of the author of the blog (most likely you)
```
Comment (nikit19) [inline _data/blogs.yml:9]: Please don't delete this

File: _data/blogs.yml [hunk 2]
```diff
@@ -3,10 +3,10 @@
...
+- url: https://myblogs938167194.wordpress.com/2018/01/01/my-google-code-in-participation/
+  title: My Google Code-In Participation
+  img: Momtahina.png
```
Comment (nikit19) [inline _data/blogs.yml:8]: Please upload this image in the img/blogs folder

**Review by nikit19 — CHANGES_REQUESTED** [review]
Please make these changes

**Comment by ParthS007** [comment]
closing this as a duplicate of #789

**Pull request closed without merging**